lib/oli_web/components/delivery/schedule_gating_assessement.ex

```diff
@@ -0,0 +1,71 @@
+defmodule OliWeb.Components.Delivery.ScheduleGatingAssessment do
+  use OliWeb, :html
+
+  attr :section_slug, :string
+  attr :uri, :string
+
+  def tabs(assigns) do
+    assigns =
+      assign(assigns,
+        active_tab: determine_tab(assigns[:uri])
+      )
+
+    ~H"""
+    <div class="w-full px-6 py-4">
```

Comment:
Can we use `container mx-auto` instead of `w-full px-6` here so that these tabs line up nicely with the rest of the
UI when the browser window is resized?
```suggestion
    <div class="container mx-auto py-4">
```